_posts/2022-05-05-8.0.0-release.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Apache Arrow 8.0.0 Release"
+date: "2022-05-05 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 8.0.0 release. This covers
+over 3 months of development work and includes [**586 resolved issues**][1]
+from [**YYY distinct contributors**][2]. See the Install Page to learn how to
+get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 7.0.0 release, Kun Liu, Raphael Taylor-Davies Xudong Wang, Yijie Shen
+and Liang-Chi Hsieh have been invited to be committers.
+Thanks for your contributions and participation in the project!
+
+## Columnar Format Notes
+
+## Arrow Flight RPC notes
+
+## C++ notes
+
+## C# notes
+
+## Go notes
+
+## Java notes
+
+## JavaScript notes
+
+## Python notes
+
```

Comment:
@jorisvandenbossche can you add the Python notes to the release blog post?

Reply:
```suggestion
* Tables and Datasets now support ``join`` operation.
* ``ParquetDataset`` is now officially deprecated
* It is now possible to create references to nested fields using ``py.field("a", "b")``
* Docstrings in ``Schema``, ``ChunkedArray``, ``Tensor``, ``RecordBatch``, ``parquet`` and ``Table`` now include examples on how to use the methods and classes
* Parquet now supports encryption
* Multiple bugfixes and a few segfault resolved.
```

Reply:
I'll do my best to add tickets references, even though some of the phrases here are actually collections of multiple tickets.